lib/src/executable/options.dart

```diff
@@ -78,6 +79,34 @@ class ExecutableOptions {
       ..addFlag('embed-source-map',
           help: 'Embed source map contents in CSS.', defaultsTo: false);
 
+    parser
+      ..addSeparator(_separator('Warnings'))
+      ..addFlag('quiet', abbr: 'q', help: "Don't print warnings.")
+      ..addFlag('quiet-deps',
+          help: "Don't print compiler warnings from dependencies.\n"
+              "Stylesheets imported through load paths count as dependencies.")
+      ..addFlag('verbose',
+          help: "Print all deprecation warnings even when they're repetitive.")
+      ..addMultiOption('fatal-deprecation',
+          help: 'Deprecations to treat as errors. You may also pass a Sass\n'
+              'version to include any behavior deprecated in or before it.\n'
+              'See https://sass-lang.com/documentation/breaking-changes for \n'
+              'a complete list.',
+          allowedHelp: {
+            for (var deprecation in Deprecation.values)
+              if (deprecation.deprecatedIn != null &&
+                  deprecation.description != null)
+                deprecation.id: deprecation.description!,
+          })
+      ..addMultiOption('future-deprecation',
+          help: 'Opt-in to a deprecation early.',
```

Comment:
Nit: "Opt in"

"Opt-in" with a hyphen is an adjective; the verb form always uses a space.

Reply:
Done